src/pages/OfferNew/Checkout/hooks.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  CompleteQuote,
+  SignState,
+  useMemberQuery,
+  useRedeemedCampaignsQuery,
+} from 'data/graphql'
+import * as React from 'react'
+import { InsuranceType } from 'utils/insuranceDomainUtils'
+import { adtraction, trackStudentkortet } from 'utils/tracking'
+
+export enum VisibilityState {
+  CLOSED,
+  CLOSING,
+  OPENING,
+  OPEN,
+}
```

Comment:
I kinda like writing enums with a value. For example now when I just tried to console.log what the `VisibilityState` was I think it's handy to see the value.  
```suggestion
export enum VisibilityState {
  CLOSED = 'CLOSED',
  CLOSING = 'CLOSING',
  OPENING = 'OPENING',
  OPEN = 'OPEN',
}
```